#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'cmd_vel_smoother'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
gen.add("x_acc_lim", double_t, 0, "", 0.1)
gen.add("y_acc_lim", double_t, 0, "", 0.1)
gen.add("yaw_acc_lim", double_t, 0, "", 0.1)
gen.add("desired_rate", double_t, 0, "", 10.0)
gen.add("decel_factor", double_t, 0, "", 0.9)
gen.add("interpolate_max_frame", int_t, 0, "", 5)

exit(gen.generate(PACKAGE, "cmd_vel_smoother", "CmdVelSmoother"))
